x-pack/test/functional/apps/ml/embeddables/anomaly_charts_dashboard_embeddables.ts

```diff
@@ -109,8 +110,52 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await PageObjects.timePicker.pauseAutoRefresh();
           await ml.dashboardEmbeddables.assertAnomalyChartsSeverityThresholdControlExists();
           await ml.dashboardEmbeddables.assertAnomalyChartsExists();
+          await PageObjects.dashboard.saveDashboard(testData.dashboardTitle);
         });
       });
     }
+
+    describe('supports migrations', function () {
+      const panelTitle = `Saved ML anomaly charts for fq_multi_1_ae`;
+      const dashboardSavedObject = {
```

Comment:
Probably worth moving the saved object configs and then looping through each item in the list, making sure it can be loaded successfully. Then for 8.0, a `7.16` config can be added with minimal changes to the code required.

Reply:
Updated here https://github.com/elastic/kibana/pull/116520/commits/eee3561c7c72a1cd85b7c65f4abc412935bd366f